lib/rules/prefer-true-attribute-shorthand.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @author Pig Fang
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const utils = require('../utils')
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description:
+        'require shorthand form attribute when `v-bind` value is `true`.',
```

Comment:
To stay consistent with other descriptions:

```suggestion
        'require shorthand form attribute when `v-bind` value is `true`',
```